src/libraries/System.Net.Http/tests/PerformanceTests/HPackHuffmanBenchmark/HPackHuffmanBenchmark.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)System\Net\Http\aspnetcore\Http2\Hpack\Huffman.cs" Link="Common\System\Net\Http\aspnetcore\Http2\Hpack\Huffman.cs" />
```

```suggestion
    <Compile Include="$(CommonPath)System\Net\Http\aspnetcore\Http2\Hpack\Huffman.cs"
             Link="Common\System\Net\Http\aspnetcore\Http2\Hpack\Huffman.cs" />
```

And other linked files similarly.
And yes, I know, VS puts it on the same line, but for readability of csproj it was decided to split the lines (see #1266 and follow up discussion in the linked PR).

---

Done